### ISSUE TITLE
Consider the possibility to add inline creation of modelling Things

### ISSUE THREAD

**Issue by naronatrhea** [RHEAGROUP/CDP4-IME-Community-Edition#96]
### Prerequisites

- [ ] I have written a descriptive issue title
- [ ] I have verified that I am running the latest version of the CDP4-IME
- [ ] I have searched [open](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
To limit the number of clics the user has to perform, it might be worth considering adding the capability to inline create a Thing for at least ElementDefinition/parameters

### Logs

  - [ ] There are no logs regarding the issue
  - { ] I have attached a log file or included the relevant section in this issue  
  
### System Configuration

- CDP4 version:
  - [ ] CDP4-IME: 
  - [ ] Other: 
  -
- Web Services:
  - [ ] CDP4 Web Services version: 
  - [ ] OCDT WSP version: 
- Environment (Operating system, version and so on):
  - [ ] Microsoft Excel (32/64 bit): 
  - [ ] Windows OS (32/64 bit): 
  - [ ] Additional information:

**Comment by samatrhea** [comment]
can you explain a bit how that would work. Could also make sense for the requirements browser.

**Comment by naronatrhea** [comment]
we could pop up an empty line within an element definition to create a parameter with a combobox supporting edit, so that the user can type in a parameter type and create it.

same would apply for ElementDefinition.

Requirement would be a bit more complicated to do because there are more the user can edit (like the text).

**Comment by samatrhea** [comment]
good idea. and then some good behavior for when the user wants to cancel without it being created on the server
